src/main/java/dk/sdu/se_f22/sharedlibrary/models/Product.java

```diff
@@ -6,64 +6,92 @@
 import java.util.HashMap;
 
 public class Product { //initialize class
-
+    
     private final HashMap<ProductAttribute, String> productAttributes; //initialize hashmap to contain product attributes
-    private final ArrayList<String> availableAt; //initialize array of strings to contain local stores with stock
-
+    private ArrayList<String> availableAt; //initialize array of strings to contain local stores with stock
```

Comment:
Couldn't the availableAt list, not just be removed entirely?
You have a seperate method for getting the locations.
So that contains the IN_STOCK attribute, which is just a comma seperated list.

So the getLocations could be something like `return new ArrayList<String>(List.of(this.get(ProductAttribute.IN_STOCK).split(",")));`

Reply:
Done